gateway-js/src/__tests__/integration/networkRequests.test.ts

```diff
@@ -158,10 +162,10 @@ it('Updates Supergraph SDL from remote storage', async () => {
   gateway.onSchemaLoadOrUpdate(schemaChangeCallback);
 
   await gateway.load(mockApolloConfig);
-  expect(gateway['compositionId']).toMatchInlineSnapshot(`"originalId-1234"`);
+  expect(gateway['compositionId']).toMatchInlineSnapshot('"originalId-1234"');
```

Comment:
I think this is going to be a problem. `prettier` is writing these snapshots under the hood and uses backticks when it does. As snapshots update, I expect the single quotes will be auto-rewritten to backticks. Is this something we can tune, or maybe something that specifically leaves inline snapshots alone?

Reply:
I'm not 100% sure what your concern is. I think the linter expects strings to be built with backticks iff there is a variable referenced. For example `\`originalId-${id}\``, but will use single quotes if there is not a variable present. But in either case, the string that we're passing to `toMatchInlineSnapshot` would be the same, no?

Reply:
Yeah, the issue is with jest/prettier rather than the linter. It's minor, but it is an issue.

If you go to update this test with a different id: https://github.com/apollographql/federation/blob/b0c3db7451d6f868e613933f8983b7648e4bf4aa/gateway-js/src/__tests__/integration/networkRequests.test.ts#L136
And run the tests with snapshot update flag `-u` this works as expected on `main`, but `prettier` (via jest) throws an error if you try to do the same thing on this branch.